frontend/src/metabase/dashboard/components/DashCard/DashCardActionsPanel/DashCardActionsPanel.tsx

```diff
@@ -103,14 +130,16 @@ export function DashCardActionsPanel({
   }
 
   if (!isLoading && !hasError) {
-    if (onReplaceAllVisualizationSettings && !disableSettingsConfig) {
+    if (!disableSettingsConfig) {
```

Comment:
Typescript complained about this check, saying that `onReplaceAllVisualizationSettings` always presents, so either the type is wrong or the check is redundant